### ISSUE TITLE
prettier-eslint-cli does not throw eslint's errors

### ISSUE THREAD

**Issue by arfa** [prettier/prettier-eslint-cli#87]
- `prettier-eslint-cli` version: 4.1.1
- `prettier-eslint` version: 6.4.2
- `prettier` version: 1.5.2
- `eslint` version: 3.19.0

`.eslintrc`:

```javascript
{
  "env": {
    "browser": true,
    "es6": true
  },
  "parser": "babel-eslint",
  "parserOptions": {
    "ecmaFeatures": {
      "jsx": true
    }
  },
  "extends": "defaults",
  "plugins": [
    "react"
  ],
  "rules": {
    "no-console": 2,
    "no-const-assign": 2,
    "no-unused-vars": 2,
    "prefer-const": 2,
    "semi": 2,
    "comma-dangle": 0,
    "react/jsx-uses-react": 2,
    "react/jsx-uses-vars": 2
  }
}
```

Problem description:

I have `"no-console": 2` in my `.eslintrc` file.
I put a `console.log` in my code so when I run the command `eslint` it throws an error.

![capture d ecran 2017-07-12 a 14 00 36](https://user-images.githubusercontent.com/729307/28116958-aac8ca8e-670b-11e7-9d3f-530a93cef83a.png)

However when I run `prettier-eslint` it doesn't show this error (I have tried all the `--log-level` possible values).

If I am not missing something, it is supposed to show the error as it is mentioned in  [prettier-eslint](https://github.com/prettier/prettier-eslint#user-content-throws)

> prettier-eslint will propagate errors when either prettier or eslint fails for one reason or another. In addition to propagating the errors, it will also log a specific message indicating what it was doing at the time of the failure.

**Comment by kentcdodds** [comment]
Hi @arfa, :wave:

That phrase in the docs is referring to errors thrown (like when parsing), not errors in the rules. `prettier-eslint` is focused only on formatting things that can be formatted, not about reporting rule violations. It is intended to be used completely separately from eslint (you should have both).

So you use `prettier-eslint` to autoformat your code, and `eslint` to find issues which cannot be autofixed.

I hope that helps.

**Comment by arfa** [comment]
Hi @kentcdodds

Thanks you for your response. it is clear now 👍 

Maybe you should specify that in the docs. 
I think it is important to know that we should have both.

**Comment by kentcdodds** [comment]
Agreed. Would you like to [makeapullrequest.com](http://makeapullrequest.com)? :smile:

**Comment by arfa** [comment]
I wil do it 😄 

It will be a PR on [prettier-eslint](https://github.com/prettier/prettier-eslint)
